web/app/js/components/Sidebar.jsx

```diff
@@ -115,19 +130,45 @@ class Sidebar extends React.Component {
     }
   }
 
+  handleNamespaceSelector(value) {
+    this.setState({namespaceFilter: value});
+  }
+
+  // Filters resources retrieved from a stat query by namespaces. Does not include resources of type "authority"
+  filterResourcesByNamespace(resources, namespace) {
+    if (namespace === "all") {
+      return _.mapValues(resources, o => {
+        return _(o)
+          .filter(r => r.added)
```

Comment:
the only difference between these is this line... what if we refactored the repeated code by putting the filter into a named variable?
```
  filterResourcesByNamespace(resources, namespace) {
    let filter = namespace === "all" ? r => r.added :
      r => r.added && r.namespace === namespace;

    return _.mapValues(resources, res => _.filter(res, filter));
  }
```

Reply:
Slick! I like it!